packages/posthog/package.json

```diff
@@ -67,7 +67,7 @@
     "@ionic/eslint-config": "0.4.0",
     "@ionic/swiftlint-config": "2.0.0",
     "eslint": "8.57.0",
-    "posthog-js": "1.160.3",
+    "posthog-js": "1.265.1",
```

Comment:
Revert this change. Otherwise, we cannot merge it before the next major release since this would be a breaking change.